src/commands/translation/push.ts

```diff
@@ -53,6 +54,8 @@ export default class TranslationPush extends BaseCommand<
   async run(): Promise<void> {
     const { flags } = this.props;
 
+    await checkTranslationsFeature(this.apiV1);
```

Comment:
We need to modify this as this shouldn't error if the feature is not enabled, instead it should **skip** pushing translations as it's not supported on your account